src/runtime_src/tools/xclbinutil/SectionBMC.h

```diff
@@ -23,23 +23,23 @@
 // --------------- C L A S S :   S e c t i o n B M C -------------------------
 class SectionBMC : public Section {
  public:
-  enum SubSection{
-    SS_UNKNOWN,
-    SS_FW,
-    SS_METADATA
+  enum class SubSection {
+    UNKNOWN,
+    FW,
+    METADATA
```

Comment:
They are not macros
```suggestion
    unknown,
    fw,
    metadata
```

Reply:
Updated the codebase to no longer use the MACRO naming convention.